agent/consul/leader_connect_ca.go

```diff
@@ -13,6 +13,8 @@ import (
 
 	"github.com/hashicorp/go-hclog"
 	uuid "github.com/hashicorp/go-uuid"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
```

Comment:
I'm not really sure if it's worth pulling this in 🤔 

For what we use it later it seems that it's redundant. I can see the full value here if Consul could support internationalization of strings but it doesn't yet. Unless you or the team feels strongly for this, I'd suggest we punt on this change 🙈 

Hopefully I didn't miss a critical use case for this 🤦🏼 🤞🏼

Reply:
Yeah, I think I was just trying to leave things as-is as much as possible. Per the Slack thread with @rboyer, I think we can just remove the use case.